src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/JiraSharedStorageResource.kt

```diff
@@ -0,0 +1,8 @@
+package com.atlassian.performance.tools.awsinfrastructure.api.jira;
+
+/**
+ * Describes a set of resources used by Jira and stored in shared storage (eg. shared home or S3).
+ */
+enum class JiraSharedStorageResource {
```

Comment:
With the recent change this is no longer used. Should we remove it?

Reply:
Absolutely, thanks.